### PR TITLE
Updates to onboarding for Amazon Sales Channel

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) updates the following sections in the TOC, with random other find and replace formatting changes.
- Amazon Sales Channel 4.0+ > Amazon Sales Channel
- Amazon Sales Channel 4.0+ > Amazon and Sales Channel Basics
- Amazon Sales Channel 4.0+ > Onboard Amazon Sales Channel
stops at "Store Settings"

## Affected documentation pages

See TOC sections listed above

## Affected Magento editions

- [x] Open Source
- [x] Commerce
- [x] B2B

## Additional information

Staging build #381 (in progress at submission)

### CODE REVIEW

File: src/sales-channels/asc/floor-price.md [hunk 1]
```diff
@@ -21,11 +21,11 @@ Floor price is only used with [intelligent repricing rules]({% link sales-channe
 
 Define your lowest price setting in the **Floor Price** section.
 
-1. For **Floor Price Source**, select an attribute from the drop-down.
+1. For **Floor Price Source**, choose an attribute from the drop-down.
```
Comment (leslietilling) [inline src/sales-channels/asc/floor-price.md]:
```suggestion
1. For **Floor Price Source**, choose an attribute.
```

File: src/sales-channels/asc/intelligent-repricing-rules.md [hunk 2]
```diff
@@ -19,15 +19,15 @@ Sections of an intelligent repricing rule include:
 
 Define the Rule Type in the **Select Price Rule Type** section.
 
-1. For **Rule Type**, select `Intelligent repricing rule` in the drop-down.
+1. For **Rule Type**, choose `Intelligent repricing rule` in the drop-down.
```
Comment (leslietilling) [inline src/sales-channels/asc/intelligent-repricing-rules.md]:
```suggestion
1. For **Rule Type**, choose `Intelligent repricing rule`.
```

File: src/sales-channels/asc/intelligent-repricing-rules.md [hunk 3]
```diff
@@ -19,15 +19,15 @@ Sections of an intelligent repricing rule include:
...
 
-1. For **Competitor Price Source**, select an option in the drop-down.
+1. For **Competitor Price Source**, choose an option in the drop-down.
```
Comment (leslietilling) [inline src/sales-channels/asc/intelligent-repricing-rules.md]:
```suggestion
1. For **Competitor Price Source**, choose an option.
```

File: src/sales-channels/asc/intelligent-repricing-rules.md [hunk 4]
```diff
@@ -19,15 +19,15 @@ Sections of an intelligent repricing rule include:
...
 
     - **Use "Buy Box" Price**: Choose when you want to adjust your Amazon pricing based on the Amazon Buy Box seller price. A [Buy Box]({% link sales-channels/asc/buy-box-competitor-pricing.md %}) price exists when multiple sellers on Amazon offer the same product. Amazon defines the Buy Box seller based on performance requirements. Merchants seek to win the Buy Box seller status and offers maximum visibility of your product listings.
 
     - **Use Lowest Competitor Price**: Choose when you want to compare and adjust your listing price to competitor pricing for the same product. When this option is selected, the **Minimum Positive Feedback** and **Minimum Feedback Count** fields enable.
 
-1. If enabled, select an option for **Minimum Positive Feedback** .
+1. If enabled, choose an option for **Minimum Positive Feedback** .
```
Comment (leslietilling) [inline src/sales-channels/asc/intelligent-repricing-rules.md]:
```suggestion
1. If enabled, choose an option for **Minimum Positive Feedback**.
```

File: src/sales-channels/asc/ob-create-listing-rule.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+1. Click <span class="btn">View Store</span> for the added store.
+
+    The store's dashboard appears with the `No products listed to Amazon` message.
+
+1. Click <span class="btn">Preview and List Eligible Products</span>.
+
+    The Listing Rules page appears.
+
+1. Define your desired conditions for the eligibility of products to be listed on Amazon and click <span class="btn">Preview changes</span>. You can also click <span class="btn">Preview changes</span> to skip this step.
```
Comment (leslietilling) [inline src/sales-channels/asc/ob-create-listing-rule.md]:
```suggestion
1. Define your desired conditions for the eligibility of products to be listed on Amazon and click <span class="btn">Preview changes</span>, or click <span class="btn">Preview changes</span> to skip this step.
```

File: src/sales-channels/asc/order-settings.md [hunk 6]
```diff
@@ -28,15 +28,15 @@ When an order is received from Amazon, the extension is set to Create Magento Or
...
 
-1. For **Import Amazon Orders Into Magento Store** (required), select which Magento Store the Amazon orders will be associated with when they are created in the Magento Order grid. The list of options in this drop-down is dependent on the Magento stores you have set up in your configuration. See [Stores]({% link stores/stores-all-stores.md %}).
+1. For **Import Amazon Orders Into Magento Store** (required), choose which Magento Store the Amazon orders will be associated with when they are created in the Magento Order grid. The list of options in this drop-down is dependent on the Magento stores you have set up in your configuration. See [Stores]({% link stores/stores-all-stores.md %}).
 
-1. For **Customer Creation**, select an option in the drop-down. Options:
+1. For **Customer Creation**, choose an option in the drop-down. Options:
```
Comment (leslietilling) [inline src/sales-channels/asc/order-settings.md]:
```suggestion
1. For **Customer Creation**, choose an option:
```

File: src/sales-channels/asc/order-settings.md [hunk 7]
```diff
@@ -28,15 +28,15 @@ When an order is received from Amazon, the extension is set to Create Magento Or
...
 
     - **No Customer Creation (guest)**: Choose when you do not want to create a customer account in Magento using the imported customer data from the Amazon order. When selected, this option tells Magento to process an imported Amazon order the same way it processes a guest checkout in Magento.
 
     - **Build New Customer Account**: Choose when you want to create a New Customer Account in Magento using the customer data imported with the Amazon order. This builds your customer database from your Amazon orders.
 
-1. For **Order Number Source**, select an option in the drop-down. Options:
+1. For **Order Number Source**, choose an option in the drop-down. Options:
```
Comment (leslietilling) [inline src/sales-channels/asc/order-settings.md]:
```suggestion
1. For **Order Number Source**, choose an option:
```

File: src/sales-channels/asc/order-settings.md [hunk 8]
```diff
@@ -45,7 +45,7 @@ When an order is received from Amazon, the extension is set to Create Magento Or
     {:.bs-callout .bs-callout-info}
     Once an order is imported, the Amazon Order Number and the Magento Order Number display in the [Orders tab]({% link sales-channels/asc/managing-orders.md %}).
 
-1. For **Pending Orders** (required), select an option in the drop-down. Options:
+1. For **Pending Orders** (required), choose an option in the drop-down. Options:
```
Comment (leslietilling) [inline src/sales-channels/asc/order-settings.md]:
```suggestion
1. For **Pending Orders** (required), choose an option:
```

File: src/sales-channels/asc/order-settings.md [hunk 9]
```diff
@@ -54,7 +54,7 @@ When an order is received from Amazon, the extension is set to Create Magento Or
     {:.bs-callout .bs-callout-info}
     Remember, Amazon orders do not import into and create new orders in Magento until Amazon verifies the order and payment method and changes the order status to `Unshipped`.
 
-1. For **Order Status** (required), select an option from the drop-down. Options:
+1. For **Order Status** (required), choose an option from the drop-down. Options:
```
Comment (leslietilling) [inline src/sales-channels/asc/order-settings.md]:
```suggestion
1. For **Order Status** (required), choose an option:
```

File: src/sales-channels/asc/standard-price-rules.md [hunk 10]
```diff
@@ -12,15 +12,15 @@ Sections of a standard price rule action include:
 
 ## To complete the Price Rule Actions section for a Standard Price Rule:
 
-1. For **Rule Type**, select `Standard price rule` in the drop-down.
+1. For **Rule Type**, choose `Standard price rule` in the drop-down.
```
Comment (leslietilling) [inline src/sales-channels/asc/standard-price-rules.md]:
```suggestion
1. For **Rule Type**, choose `Standard price rule`.
```

File: src/sales-channels/asc/standard-price-rules.md [hunk 11]
```diff
@@ -12,15 +12,15 @@ Sections of a standard price rule action include:
...
 
-1. For **Price Action**, select an option in the drop-down.
+1. For **Price Action**, choose an option in the drop-down.
```
Comment (leslietilling) [inline src/sales-channels/asc/standard-price-rules.md]:
```suggestion
1. For **Price Action**, choose an option.
```

**Comment by hickskenh** [comment]
running tests

**Pull request merged**